corehq/apps/geospatial/reports.py

```diff
@@ -68,7 +68,8 @@ def template_context(self):
                 {'id': p.id, 'name': p.name, 'geo_json': p.geo_json}
                 for p in GeoPolygon.objects.filter(domain=self.domain).all()
             ],
-            'es_indexing_message': celery_task_tracker.get_message()
+            'es_indexing_message': celery_task_tracker.get_message(),
+            'is_error_es_index_message': celery_task_tracker.is_error(),
```

Comment:
I'm going to be very nitpicky here: 
How about nesting the message-related data under a "message" key or similar? I like to think of it at "packaging" it nicely, otherwise it feels like it's "lying around all over the place" in the context. 

So it would become something like:
```
'task_message': {'message': ..., 'is_error': ...}
```

As you can also see, the flag name showing whether it's an error or not is also now much shorter, since it's already in the context of the task message and you don't need to convey the context in the variable name itself.

Reply:
I like this idea. It keeps things more contained and organized which is nice. I could even take this a step further and create a `get_status()` function in the `CeleryTaskTracker` class to return a dict in this format. We would then only need to call a single function for both the views and reports.

Reply:
Addressed in [6906d82](https://github.com/dimagi/commcare-hq/pull/35126/commits/6906d828c78293671103f8edadef77d568694369).